Arrangement_on_surface_2/include/CGAL/Arr_dcel_base.h

```diff
@@ -91,6 +91,9 @@ template <typename Point_> class Arr_vertex_base {
   /*! Destructor. */
   virtual ~Arr_vertex_base() {}
```

Comment:
I know this is not your code, but seeing this destructor I am wondering if polymorphism is really used here ? 
If not (this must be checked), virtual methods can be avoided in order to save memory.